lib/fizzy/execute.hpp

```diff
@@ -100,6 +100,20 @@ execution_result execute(
 // TODO: remove this helper
 execution_result execute(const Module& module, FuncIdx func_idx, std::vector<uint64_t> args);
 
+// Function that should be used by instantiate as imports, identified by module and function name.
+struct ImportedFunction
```

Comment:
I'm not very happy with the name, maybe `NamedFunctionImport` or `NamedFunction`?